lib/rspec/mocks/any_instance/message_chains.rb

```diff
@@ -49,7 +49,9 @@ def all_expectations_fulfilled?
         # @private
         def unfulfilled_expectations
           @chains_by_method_name.map do |method_name, chains|
-            method_name.to_s if ExpectationChain === chains.last unless chains.last.expectation_fulfilled?
+            unless chains.empty? || chains.last.expectation_fulfilled?
```

Comment:
I wonder, what if we check `ExpectationChain === chains.last` first?
If `chains` is empty, `ExpectationChain === chains.last` would return `false`.
Can this be simplified to?
```ruby
method_name.to_s if ExpectationChain === chains.last && !chains.last.expectation_fulfilled?
```